packages/lib/src/components/internal/OpenInvoice/OpenInvoice.tsx

```diff
@@ -297,8 +296,10 @@ export default function OpenInvoice(props: OpenInvoiceProps) {
 
             {activeFieldsets.deliveryAddress && (
                 <Address
-                    allowedCountries={props.allowedCountries}
+                    allowedCountries={props?.deliveryAddressSpecification?.allowedCountries ?? props.allowedCountries}
                     countryCode={countryCode}
+                    requiredFields={props.deliveryAddressRequiredFields}
+                    specifications={props.deliveryAddressSpecification}
```

Comment:
Did you test these changes across other open invoice payment methods too?

Reply:
Yes tested manually, it should be good. If the `props` doesn't contain `deliveryAddressRequiredFields` or `deliveryAddressSpecification`, the Address component takes the default values.